docs/content/Gameplay/Central-Monitor.md

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
Please run the MKDocs Serve action and actually validate if the docs look as you intend for them to before making PRs to docs /notneg :)
For example, the list looks like this:
<img width="902" height="461" alt="image" src="https://github.com/user-attachments/assets/1e537c45-dd10-4b5c-821c-76c307044eb9" />